docs/content/commands/update.md

```diff
@@ -65,3 +65,10 @@ If you want to update a single group you can use the following command:
   `--redirects`: Creates binding redirects for the NuGet packages.
 
   `--no-install`: Skips paket install --hard process afterward generation of [`paket.lock` file](lock-file.html).
+
+## Updating http dependencies
+
+The update command has no effect on http dependencies. If you want to update a file you need to use the [`paket install` command](paket-install.html) with `--force [-f]` option.
```

Comment:
Sorry, always did it through `paket install`. When I manually edit `paket.dependencies` and then execute `paket install` - i feel that everything is under control ;)